deploy/install-driver.sh

```diff
@@ -16,6 +16,13 @@
 
 set -euo pipefail
 
+# min version of this nfs plugin supporting CSI snapshots
+snap_ver='4.3.0'
```

Comment:
kind of expecting the next release to be `v4.3.0` to install the external snapshot-controller and CRDs for `master` and `v4.3.0` and higher.